cmd/cluster/cpd.go

```diff
@@ -112,7 +114,9 @@ func (o *cpdOptions) run() error {
 				return fmt.Errorf("subnet %s does not have a default route to 0.0.0.0/0\n Run the following to send a SerivceLog:\n osdctl servicelog post %s -t https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/aws/InstallFailed_NoRouteToInternet.json", subnet, o.clusterID)
 			}
 		}
-		fmt.Printf("Next step: run the verifier egress test: osdctl network verify-egress --cluster-id %s\n", o.clusterID)
+		fmt.Printf("Attempting to run: osdctl network verify-egress --cluster-id %s\n", o.clusterID)
+		ev := &network.EgressVerification{ClusterId: o.clusterID}
```

Comment:
**thought:** Even if the cluster is BYOVPC non-privatelink, the current error message coming from the verification should should be enough to prompt the user to manually re-run the egress verification with the right parameters.